### DISCUSSION TITLE
There is way of getting csv or any other table format output from zircolite ?

### DISCUSSION THREAD

**Discussion by NofLevi** [wagga40/Zircolite#39] [Q&A] [answered]
same as title

**Comment by wagga40** [comment, marked as answer]
Hi, 

**YES ! you can have it using the `--csv` option.** 

Be careful, CSV is not very well suited for EVTX logs.

You can also customise your output with the templating system, check the docs : [here](https://github.com/wagga40/Zircolite/blob/master/docs/Advanced.md#templating-and-formatting).

**Reply by NofLevi** [reply]
Thanks for quick and informative reply.
